_data/events_gdcr2018/Pune-India-CDKGlobal.json

```diff
@@ -0,0 +1,19 @@
+{
+  "title": "Coderetreat @ CDK Global (India) Pvt. Ltd. Pune",
+  "url": "https://sites.google.com/view/cdkpuneevents",
+  "moderators": [
+    "Vibhor Mamgain",
+    "Deepesh Patel",
+    "@patel_deepesh"
```

Comment:
Are we here talking about the same person? If yes, preferrably use only your twitter handle instead of both your full name and twitter handle.